### PR TITLE
Add support for new LRO implementation

### PR DESCRIPTION
**Pull request by vam-google**
https://github.com/googleapis/gax-java/issues/147
https://github.com/googleapis/gax-java/issues/146

### CODE REVIEW

File: google-cloud-speech/src/main/java/com/google/cloud/speech/v1/SpeechSettings.java [hunk 1]
```diff
@@ -77,7 +78,7 @@
   private static final String DEFAULT_GAPIC_NAME = "gapic";
   private static final String DEFAULT_GAPIC_VERSION = "";
 
-  private static final String PROPERTIES_FILE = "/com/google/cloud/speech/project.properties";
+  private static final String PROPERTIES_FILE = "/com/google/cloud/speech/v1/project.properties";
```
Comment (garrettjonesgoogle) [inline google-cloud-speech/src/main/java/com/google/cloud/speech/v1/SpeechSettings.java]: I don't think this should be changing

**Comment by garrettjonesgoogle** [comment]
Looks good for starting the change rollout (gax, toolkit)

**Comment by vam-google** [comment]
Using new released gax version now (compilable PR now). PTAL.

**Comment by garrettjonesgoogle** [comment]
LGTM

**Pull request merged**